ADD: recyclerView,ListAdapter with Flow,LiveData

014967 opened the pull request:
온전히 Flow만 사용하는 것은 잘모르겠어서 LiveData와 함께 사용해서 더미데이터로 테스트를 해보았습니다
더미데이터 클래스는 나중에 db 보면서 바꿀려고 합니당

014967 committed:
ADD: recyclerView,ListAdapter with Flow,LiveData

paksuua committed:
Update: build.gradle

사용하지 않는 라이브러리 제거

paksuua commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -11,15 +11,15 @@
 
     <com.google.android.material.bottomnavigation.BottomNavigationView
-        android:id="@+id/btnv_view"
+        android:id="@+id/btn_view"
```
btn은 <Button>에서 쓰는 약어입니다. BottomNavigationView는 btnv로 써주세요

paksuua commented on app/src/main/res/layout/item_chatting_room_list_two.xml:
```diff
@@ -0,0 +1,133 @@
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:text="ㅁㄴㅇㄹㅁㄴㅇㄹㅁㄴㅇ"
```
UI 테스트용 데이터는 
android:text=""에 넣으면 안됩니다.
tools:text=""에 넣으면 xml preview로 확인 가능하고 실질적으로 실행했을때 더미데이터를 넣고 싶으시면 데이터클래스에서 더미데이터를 만드는게 더 좋습니다.

014967 replied:
확인했습니다

paksuua commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -11,15 +11,15 @@
 
     <com.google.android.material.bottomnavigation.BottomNavigationView
-        android:id="@+id/btnv_view"
+        android:id="@+id/btn_view"
```
btn은 Button의 타입 이름입니다.
btnv에서 왜 변경하셨는지 모르겠습니다.

014967 replied:
이런 오타인줄 알았네요

paksuua commented on app/src/main/java/com/sgs/devcamp2/flametalk_android/ui/MainActivity.kt:
```diff
@@ -28,13 +29,13 @@ class MainActivity : AppCompatActivity() {
         navController.addOnDestinationChangedListener { _, destination, _ ->
             if (destination.id == R.id.navigation_signin || destination.id == R.id.navigation_signup) {
-                binding.btnvView.visibility = View.GONE
+                binding.btnView.visibility = View.GONE
```
BottomNaviagationView 타입 약자 btn 쓰시면 안됩니다. btnv로 해주세요.

paksuua commented on app/src/main/java/com/sgs/devcamp2/flametalk_android/ui/chatlist/ChatListAdapter.kt:
```diff
@@ -0,0 +1,119 @@
+import com.sgs.devcamp2.flametalk_android.domain.model.response.chatlist.ChattingRoom
+
+class ChatListAdapter : ListAdapter<ChattingRoom, RecyclerView.ViewHolder>(diffUtil) {
+// 클래스에서만 한번만 쓸려고하는 것
```
클래스의 역할을 이해할 수 있도록 주석 작성 부탁드립니다.

paksuua reviewed (changes requested):
첫 풀리퀘 축하드려요! 남긴 코멘트 한번씩 확인 후 수정해주세요

014967 committed:
FIX: Resource naming, ADD: Item Click Dialog, ChatRoomFragment 생성

1)naming convention 적용
2)recyclerview item click listener 설정 후 dialog display
3)ChatRoomFragment 생성

014967 committed:
Merge branch 'feature/chat_list_page' of https://github.com/DevCamp2Flame/FlameTalk_Android into feature/chat_list_page

014967 committed:
ADD : Comment from ChatList

ChatListFragment, ChatListViewModel, ChatListAdapter 주석 추가

paksuua reviewed (approved):
확인했습니다!
